backend/database/actionsWithTables/productActions.js

```diff
@@ -1,40 +1,63 @@
 const { Product } = require('../tables/product');
+const { Subcategorie } = require('../tables/subcategorie');
+const { Producer } = require('../tables/producer');
+
+const {
+    addNewProductAmount,
+    returnProductAmounts,
+} = require('./productAmountActions');
 
 const addNewProduct = async (data) => {
-    try {
-        return await Product.create({
-            name: data.name,
-            price: data.price,
-            unit: data.unit,
-            subcategory_id: data.subcategory_id,
-            producer_id: data.producer_id,
-        });
-    } catch (error) {
-        console.log(error);
+    const newProduct = await Product.create({
+        name: data.name,
+        price: data.price,
+        unit: data.unit,
+        subcategorie_id: data.subcategorie_id,
+        producer_id: data.producer_id,
+    });
+    let dataForProductAmount = {
+        amount: data.amount,
+        storage_id: data.storage_id,
+        product_id: newProduct.product_id,
     }
+    addNewProductAmount(dataForProductAmount);
 };
 
-const returnProducts = async () => {
+const getAllProducts = async () => {
     const products = await Product.findAll({
-        attributes: ['product_id', 'name', 'price', 'unit', 'subcategory_id', 'producer_id', 'is_deleted'],
+        attributes: ['product_id', 'name', 'price', 'unit', 'is_deleted'],
         where: {
             is_deleted: false,
-        }
+        },
+        include: [
+            {
+                model: Subcategorie,
+                where: {
+                    is_deleted: false,
+                },
+            },
+            {
+                model: Producer,
+                where: {
+                    is_deleted: false,
+                },
+            },
+        ],
     });
     return await products;
 };
 
-const isDeleteProduct = async (data) => {
+const deleteProduct = async (data) => {
```

Comment:
Still better to provide id instead of data
```suggestion
const deleteProduct = async (id) => {
```